cli/internal/daemonclient/daemonclient.go

```diff
@@ -0,0 +1,53 @@
+// Package daemonclient is a wrapper around a grpc client
+// to talk to turbod
+package daemonclient
```

Comment:
Generally design-wise I quite like that these are all thin and basically shuttle information around. We should stick to that pattern.